client/app/components/groupDetail/groupDetail.controller.js

```diff
@@ -1,6 +1,13 @@
 class GroupDetailController {
-  constructor(Group, CurrentGroup, $state, $stateParams) {
+  constructor(Group, CurrentGroup, $mdMedia, $scope, $state, $stateParams) {
     "ngInject";
+    $scope.screenIsSmall = !$mdMedia("gt-sm");
```

Comment:
Do you think it makes sense to create a global variable? Or do you mean creating a service that basically just returns $mdMedia("gt-sm")?